goodgame-application/src/main/kotlin/people/songpagu/goodgame/application/member/privacy/service/MemberPrivacyFindService.kt

```diff
@@ -0,0 +1,20 @@
+package people.songpagu.goodgame.application.member.privacy.service
+
+import people.songpagu.goodgame.application.member.privacy.incoming.MemberPrivacyFindUseCase
+import people.songpagu.goodgame.application.member.privacy.incoming.MemberPrivacyFindUseCase.MemberPrivacyAnswer
+import people.songpagu.goodgame.application.member.privacy.outgoing.MemberPrivacyFindPort
+import people.songpagu.goodgame.domain.member.model.MemberPrivacy
+
+class MemberPrivacyFindService(
+    private val memberPrivacyFindPort: MemberPrivacyFindPort
+) : MemberPrivacyFindUseCase {
+    override fun findMemberPrivacyBy(memberNumber: String): MemberPrivacyAnswer {
+        val memberPrivacy: MemberPrivacy? = memberPrivacyFindPort.findBy(memberNumber)
+        return MemberPrivacyAnswer(
+            memberNumber = memberPrivacy!!.memberNumber,
```

Comment:
아래처럼 `requireNotNull` 사용하면 돼!
```suggestion
            memberNumber = requireNotNull(memberPrivacy).memberNumber,
```

Reply:
`!!`를 써도 되긴해!
그리고 자바의 `orElseThrow()` 쓰고 싶으면

`memberPrivacy?.memberNumber?:throw RuntimeException()`
이런식으로 쓸 수 있어!

Reply:
그리고 `MemberPrivacy`가 `null`일 수 없다면 `MemberPrivacyFindPort`가 `MemberPrivacy?`를 반환하지 말고 `MemberPrivacy`를 반환하게 스펙을 정해두고 `JpaFacade`쪽에서 `null`이면 exception throw하게 하는게 더 자연스러울 것 같아!